src/index.html

```diff
@@ -5,9 +5,63 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card" >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__description">
+        Product code: 195434
+      </p>
+
+      <div class="card__product-rating">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price-container">
+        <h3 class="card__price-text">Price:</h3>
+        <p class="card__price-value">$2,199</p>
+      </div>
+
+
+      <a
+        href="#"
+        class="card__button-buy"
+        type="button"
+        data-qa="hover"
+      >
+      Buy
+      </a>
+
+    </div>
```

Comment:
```suggestion
      <a
        href="#"
        class="card__button-buy"
        type="button"
        data-qa="hover"
      >
        Buy
      </a>
    </div>
```
Remove unnecessary blank lines